<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { 
    background-color: #121212; 
    color: #E0E0E0; 
    font-family: &quot;Segoe UI&quot;; 
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1F1F1F;
color: white;
border-radius: 8px;
</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>902</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>410</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #2A2A2A;
    color: white;
    border: none;
    border-radius: 15px;
    padding-left: 15px;
    height: 30px;
}
QLineEdit:focus {
    border: 1px solid #1DB954;
}
</string>
       </property>
       <property name="placeholderText">
        <string>Search </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 25px;   /* 50px / 2 = tr√≤n */
    background-color: #666;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/avatar.jpg</normaloff>:/images/avatar.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>1280</width>
     <height>67</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>67</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>65</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1280</width>
     <height>571</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>571</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>571</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="Sidebar">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: transparent; 
    color: #E0E0E0;
    border: none;
    font-size: 16px;
    text-align: left; 
    padding: 10px 15px;
}

QPushButton:checked {
    font-weight: bold;     
    color: #00BFA5;     
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="logout">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>510</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>üîí ƒêƒÉng xu·∫•t</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>150</width>
         <height>121</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#labelLogo {
    color: #00BFA5;      
    font-weight: bold;     
    font-family: &quot;Segoe UI Semibold&quot;; 
    font-size: 18pt;
    text-align: center;
}</string>
       </property>
       <property name="text">
        <string>üéµ MUSIC üéµ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>172</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="home">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>üè† Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playlist">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>üéµ Playlist</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="MainArea" native="true">
      <widget class="QStackedWidget" name="stackedWidgetMain">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1095</width>
         <height>555</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1095</width>
         <height>555</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="homePage">
        <widget class="QScrollArea" name="scrollArea">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>0</y>
           <width>1101</width>
           <height>541</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>541</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1085</width>
            <height>1600</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1600</height>
           </size>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="playlistPage"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
